app/views/step_by_step_pages/show.html.erb

```diff
@@ -21,12 +21,12 @@
     <%= render "govuk_publishing_components/components/summary_list", {
       borderless: true,
       title: "Content",
-      edit: {
+      edit: @step_by_step_page.can_be_edited? ? {
```

Comment:
I think this logic should be moved to a method in the presenter and can be simplified. Rather than setting `edit` to `nil` it can be excluded completely:

```
    summary_list_params = {
      borderless: true,
      title: "Content",
      items: [
         .
         .
         .
      ]
    }

    if @step_by_step_page.can_be_edited?
      summary_list_params.merge!(
        edit: {
          href: edit_step_by_step_page_path(@step_by_step_page),
          data_attributes: {
            gtm: "edit-title-summary-body"
          }
        }
      )
    end
```

Then the component would only need to call:
`<%= render "govuk_publishing_components/components/summary_list", @step_by_step_page_presenter.summary_list_params %>`